core/domain/state_domain.py

```diff
@@ -1233,7 +1309,9 @@ def from_customization_arg_dict(cls, ca_dict, ca_schema):
             InteractionCustomizationArg. The customization argument domain
             object.
         """
-        def convert_content_to_domain_obj(ca_value, schema_obj_type):
+        def convert_content_to_domain_obj(
+            ca_value: Dict[str, str], schema_obj_type: str
+        ) -> Union[SubtitledHtml, SubtitledUnicode]:
```

Comment:
Can this be overloaded on `schema_obj_type`?

Reply:
Done wherever it is applicable!